src/jetson/models/utils/box_utils.py

```diff
@@ -143,6 +167,34 @@ def encode(matched:torch.Tensor, priors:torch.Tensor, variances:'list[float]'):
     return torch.cat([g_cxcy, g_wh], 1)  # [num_priors,4]
 
 
+def encode_landm(matched, priors, variances):
+    """Encode the variances from the priorbox layers into the ground truth boxes
+    we have matched (based on jaccard overlap) with the prior boxes.
+    Args:
+        matched: (tensor) Coords of ground truth for each prior in point-form
+            Shape: [num_priors, 10].
+        priors: (tensor) Prior boxes in center-offset form
+            Shape: [num_priors,4].
+        variances: (list[float]) Variances of priorboxes
+    Return:
+        encoded landm (tensor), Shape: [num_priors, 10]
+    """
+
+    # dist b/t match center and prior's center
+    matched = torch.reshape(matched, (matched.size(0), 5, 2))
+    priors_cx = priors[:, 0].unsqueeze(1).expand(matched.size(0), 5).unsqueeze(2)
+    priors_cy = priors[:, 1].unsqueeze(1).expand(matched.size(0), 5).unsqueeze(2)
+    priors_w = priors[:, 2].unsqueeze(1).expand(matched.size(0), 5).unsqueeze(2)
+    priors_h = priors[:, 3].unsqueeze(1).expand(matched.size(0), 5).unsqueeze(2)
+    priors = torch.cat([priors_cx, priors_cy, priors_w, priors_h], dim=2)
```

Comment:
based on how this code looks I think it would look much cleaner with a loop. just because its 4 cases doesn't mean its okay to write the same line 4 times.

example refactor:
```python
prior_center: List[Any] = [] # I don't know the type here but I'm sure you do. 
for index in range(0, 4):
    prior_center.append(priors[:, index].unsqueeze(1).expand(matched.size(0), 5).unsqueeze(2))

priors = torch.cat(prior_center, dim=2)
```

see how much cleaner that is? and much more readable too